### PR TITLE
Add 1.17 blog post

### LINKED ISSUE #5975: 1.17 Release
- **Prepare**
  - [ ] Complete [1.17 milestone items](https://github.com/prettier/prettier/milestone/27)
  - [ ] Run on other projects to check for regressions
  - [ ] Write release notes (#6052)
    - add a Markdown file named <code>*YYYY*-*MM*-*DD*-*version*.md</code> in [`website/blog`](https://github.com/prettier/prettier/tree/master/website/blog)
    - [prs-merged-since](https://npm.im/prs-merged-since) may be useful
- **Publish**
  - [ ] Run [release script](https://github.com/prettier/prettier/tree/master/scripts/release)
- **Postpublish**
  - [ ] Merge release notes
  - [ ] Wait for release notes to deploy
  - [ ] Tweet from [`@PrettierCode`](https://twitter.com/PrettierCode) account via TweetDeck
  - [ ] Close milestone
  - [ ] Send schema PR generated by `scripts/generate-schema.js` to [SchemaStore](https://github.com/SchemaStore/schemastore/blob/master/src/schemas/json/prettierrc.json) if we added a new option.
  - [ ] Update the [release checklist](https://github.com/prettier/prettier/wiki/Release-Checklist) with possible changes

Then monitor [Twitter](https://twitter.com/search?q=%40PrettierCode&src=typed_query&f=live)/[GitHub](https://github.com/prettier/prettier/issues) for any immediate issues 😅

### PR DESCRIPTION
**Pull request by duailibe**
Sorry this took a while, please share your feedback, I should publish this release tomorrow morning.

Closes #5975

[Preview](https://deploy-preview-6052--prettier.netlify.com/blog/2019/04/12/1.17.0.html)

### CODE REVIEW

File: website/blog/2019-04-12-1.17.0.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+
+// Output (Prettier stable)
+a(?.b[c]).c();
+let value = /** @type {string} */ this(.members[0]).functionCall();
+
+// Output (Prettier master)
+(a?.b[c]).c();
+let value = /** @type {string} */ (this.members[0]).functionCall();
+```
+
```
Comment (thorn0) [inline website/blog/2019-04-12-1.17.0.md]: "### Markdown" is missing

Comment (duailibe) [reply website/blog/2019-04-12-1.17.0.md]: Thanks

File: website/blog/2019-04-12-1.17.0.md [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+nine,
+ten,
+
+eleven
+
+) {}
+
+// Output (Prettier stable)
+function foo(
+one,
```
Comment (ikatyang) [inline website/blog/2019-04-12-1.17.0.md]: Indentations are missing.

File: website/blog/2019-04-12-1.17.0.md [hunk 3]
```diff
@@ -0,0 +1,361 @@
+---
+author: Lucas Duailibe (@duailibe)
+authorURL: https://github.com/duailibe
+title: "Prettier 1.17: More quotes options and support for shared configs"
+---
+
+This release improves HTML formatting and contains better CRLF handling, new
```
Comment (ikatyang) [inline website/blog/2019-04-12-1.17.0.md]: It seems you forgot to remove this line.

File: website/blog/2019-04-12-1.17.0.md [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
+seven,
+eight,
+nine,
+ten,
+
+eleven
+
+) {}
+
+// Output (Prettier stable)
```
Comment (ikatyang) [inline website/blog/2019-04-12-1.17.0.md]:
```suggestion
// Output (Prettier 1.16)
```

File: website/blog/2019-04-12-1.17.0.md [hunk 5]
```diff
@@ -0,0 +1,361 @@
...
+five,
+six,
+seven,
+eight,
+nine,
+ten,
+eleven
+) {}
+
+// Output (Prettier master)
```
Comment (ikatyang) [inline website/blog/2019-04-12-1.17.0.md]:
```suggestion
// Output (Prettier 1.17)
```

File: website/blog/2019-04-12-1.17.0.md [hunk 6]
```diff
@@ -0,0 +1,361 @@
...
+>   ...require("@company/prettier-config"),
+>   semi: false
+> };
+> ```
+
+## General
+
+### JavaScript
+
+#### Respect newlines between parameters ([#5260] by [@evilebottnawi])
```
Comment (ikatyang) [inline website/blog/2019-04-12-1.17.0.md]:
```suggestion
#### Respect newlines between parameters ([#5836] by [@evilebottnawi])
```

PR: #5836
Issue: #5260

**Comment by duailibe** [comment]
Thanks @ikatyang

**Pull request merged**